qiskit/pulse/instructions/play.py

```diff
@@ -93,6 +93,20 @@ def _initialize_parameter_table(self,
                         # Table maps parameter to operand index, 0 for ``pulse``
                         self._parameter_table[param].append(0)
 
+    @property
+    def parameters(self) -> Set:
+        """Parameters which determine the instruction behavior."""
+        parameters = set()
+        for pulse_param_expr in self.pulse.parameters.values():
+            if isinstance(pulse_param_expr, ParameterExpression):
+                for pulse_param in pulse_param_expr.parameters:
+                    parameters.add(pulse_param)
+        if self.channel.is_parameterized():
+            for ch_param in self.channel.parameters:
```

Comment:
It seems that the tests need to be extended with parametrised channels:  https://coveralls.io/builds/39061954/source?filename=qiskit%2Fpulse%2Finstructions%2Fplay.py#L105